project_template/src/operations.c

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#include "operations.h"

```

Comment:
If this is intending to include the public header, this should `#include <az/exampleshortname/operations.h>`.

Reply:
the reason to allow this is that it lets private headers become public without changing their source files. I like this style, but we can say "just use full name"